docs/aerodromes/Woomera.md

```diff
@@ -0,0 +1,44 @@
+---
+  title: Woomera (YPWR)
+---
+
+--8<-- "includes/abbreviations.md"
+
+## Positions
+
+| Name               | Callsign       | Frequency        | Login Identifier              |
+| ------------------ | -------------- | ---------------- | --------------------------------------|
+| Woomera ADC    | Woomera Tower  | 118.300         | WR_TWR        |
+| Woomera ATIS    |   | 118.100         | YPWR_ATIS       |
+
+## Airspace
+
+By default, WR ADC owns the **R222F** Restricted Area. This airspace has vertical limits of `SFC` to `F125`. When WR ADC is active, this airspace is classfied as Class D.
+
+Refer to [Class D Tower Skills](../../controller-skills/classdtwr) for more information.
+
+## Surveillance
+
+WR TWR is permitted to use Surveillance standards for separation. Surveillance coverage can be expected to be available at all levels within R222F.  
+For simulation purposes, visual separation is assumed to exist below the cloud base, and within 5nm. Visual separation can still be used to separate from aircraft on an instrument approach, below the cloud base.
+
+## Coordination
+### WRA
```

Comment:
Sorry I may have been misleading with a previous comment, this should be "ASP(WRA)" in the headings, but just "WRA" in the examples. Cheers